src/Core/Manifest/ManifestFileFinder.php

```diff
@@ -73,6 +74,14 @@ public function acceptDir($basename, $pathname, $depth)
             return false;
         }
 
+        // Skip if test dir inside vendor module with unexpected CI library
+        if ($depth > 3 && $basename === self::TESTS_DIR && $ignoreCILib = $this->getOption('ignore_ci_library')) {
```

Comment:
`$depth > 3` is there to make sure that the logic only applies to vendor module. (Technically, you can still use create module and stick them in you project root like in SS3)